database/migrations/2024_04_24_165732_create_languages_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
```

Comment:
I'll anticipate a comment saying that these kind or comments are no needed ;)

Reply:
Removed unnecessary comments :+1: